### DISCUSSION TITLE
where is markdown?

### DISCUSSION THREAD

**Discussion by rindy8707** [apache/superset#23858] [Q&A / Help]
I Installed superset v2.1.0

But I can't find "Markdown". (In v2.01, there was a Markdown menu on the edit dashboard - layout elements)

--> I'm looking for Markdown because I want to add a hyperlink on the dashboard. Is there any other way to do this?

**Comment by sebastianliebscher** [comment]
In Superset 2.1.0 you can use Markdown in the "Text" Layout Element, like:

```
# ✨Header 1
## ✨Header 2
### ✨Header 3

<br />

Click here to learn more about [markdown formatting](https://bit.ly/1dQOfRK)
```

**Comment by mistercrunch** [comment]
It feels like we should make it more clear/discoverable that we support Text / Markdown
